package.json

```diff
@@ -50,6 +54,8 @@
     "build": "react-scripts build",
     "dev": "cat .env.common-local .env.dev > .env.local && yarn start-dev",
     "eject": "react-scripts eject",
+    "eslint": "eslint --ext .js --ext tsx src/",
```

Comment:
I thought more about this. I think we do need to include `.ts` files for declarations.

Reply:
I just noticed this. Why is it `.js` (with a dot) and `tsx` (without a dot)?

Reply:
fixed, sorry